examples/embodiment/config/maniskill_ppo_openpi.yaml

```diff
@@ -0,0 +1,196 @@
+    queue_name: "obs_buffer"
+    queue_size: 0
+  enable_offload: False
+  # Override the default values in env/train or env/eval
+  train:
+    total_num_envs: 320
+    auto_reset: True
+    use_fixed_reset_state_ids: False
+    max_episode_steps: 80 # max episode steps for truncation
+    max_steps_per_rollout_epoch: 80
+    init_params:
+      use_multiple_plates: False
+  eval:
+    total_num_envs: 320
+    auto_reset: True
+    ignore_terminations: True
+    max_episode_steps: 80
+    max_steps_per_rollout_epoch: 80
+    use_fixed_reset_state_ids: True
+    group_size: ${algorithm.group_size}
+    is_eval: True
+    video_cfg:
+      save_video: True
+      video_base_dir: ${runner.logger.log_path}/video/eval
+    init_params:
+      use_multiple_plates: False
+
+rollout:
+  group_name: "RolloutGroup"
+  channel:
+    name: ${env.channel.name}
+    queue_name: "action_buffer"
+    queue_size: 0
+  mode: "colocate"
+  generation_backend: "huggingface"
+  recompute_logprobs: False
+  unnorm_key: libero_10
+  enable_offload: False
+  pipeline_stage_num: 1
+
+  model:
+    model_path: "/path/to/model/RLinf-Pi0-SFT"
+    precision: ${actor.model.precision}
+actor:
+  group_name: "ActorGroup"
+  channel:
+    name: ${env.channel.name}
+    queue_name: "replay_buffer"
+    queue_size: 0
+  training_backend: "fsdp"
+  micro_batch_size: 32      # how many samples each GPU processes per sampling step
+  global_batch_size: 5120   # after how many sampled examples we perform one loss backward pass. Requirement: global bz must be '_world_size' multiples of local bs.
+  seed: 0
+  enable_offload: False
+  # Override the default values in model/pi0
+  model:
+    model_name: "openpi"
+    precision: null    # bf16
+    num_action_chunks: 5 # interface for the env
+    action_dim: 7
+    is_lora: False
+    lora_rank: 32
+    gradient_checkpointing: False
+    use_wrist_image: False
+    use_proprio: True
+    num_steps: 4
+    add_value_head: True
+    # openpi specific parameters
+    openpi:
+      config_name: "pi0_maniskill"
+      num_images_in_input: 1
+      noise_level: 0.5
+      action_chunk: ${actor.model.num_action_chunks}
+      num_steps: ${actor.model.num_steps}
+      train_expert_only: True
+      action_env_dim: ${actor.model.action_dim}
+      noise_method: "flow_noise"
+      action_horizon: 8
+      noise_params: [0.16, 0.12, 200] # Tonghe appended. noise_start, noise_end, anneal_steps
+      detach_critic_input: True
+      joint_logprob: True # False for flow_sde, True for flow_noise     
+      value_after_vlm: False      # pi0: False. pi0.5: True
+    policy_setup: widowx_bridge
+
+  optim:
+    lr: 7.91e-6
+    value_lr: 1.55e-4
+    adam_beta1: 0.9
+    adam_beta2: 0.95
+    adam_eps: 1.0e-05
+    clip_grad: 1.0
+    critic_warmup_steps: 0
+
+  # Override the default values in training_backend/fsdp
+  fsdp_config:
+    strategy: "fsdp"
+    sharding_strategy: "no_shard"   # Tonghe revised from full_shard
```

Comment:
YAML config is user-facing and we wish to avoid any comments not helpful to users.